packages/react-charts/src/victory/components/ChartLegend/examples/ChartLegendTooltips.tsx

```diff
@@ -0,0 +1,51 @@
+import { ChartLabel, ChartLegend, ChartPie, ChartThemeColor } from '@patternfly/react-charts/victory';
+import { Tooltip } from '@patternfly/react-core';
+
+interface PetData {
+  x: string;
+  y: number;
+}
+
+export const ChartLegendTooltips: React.FunctionComponent = () => {
+  const data: PetData[] = [
+    { x: 'Cats', y: 35 },
+    { x: 'Dogs', y: 55 },
+    { x: 'Birds', y: 10 }
+  ];
+  const ref = React.useRef(null);
```

Comment:
We should be able to just use `useRef` rather than `React.useRef`, we'd just want to import useRef in this file.